backend/app/core/config.py

```
@@ -79,7 +79,7 @@ class Config:
 print("PROJECT_TITLE_AFTERWARDS:", settings.PROJECT_TITLE)
 
 
-async def validation_exception_handler(request: Request, exc: RequestValidationError):
+async def validation_exception_handler(exc: RequestValidationError):
```

Ruff (0.8.2)

Missing return type annotation for public function `validation_exception_handler`

(ANN201)